### ISSUE TITLE
Feature request: expose total # of parts in multi-part upload

### ISSUE THREAD

**Issue by bdelamotte** [TTLabs/EvaporateJS#262]
This library is awesome, but I was hoping to find an easy way to get the total number of parts in a multi-part upload. On the AWS server side, you can only see how many parts have been uploaded, but not how many there should be.

I would like to have an easy way to get the total number of parts that EvaporateJS creates to upload the file. I could then use this info by pushing it up to a DynamoDB table and then creating a Lambda function to get the upload status from the Server's perspective. This would be useful because you could have another function that uses Amazon SNS or SES to message someone if the upload fails after X minutes Y hours.  I would like to do this, because if someone walks away from the browser after starting to upload a huge file, I could message them if the upload fails after a certain time. I know that EvaporateJS retries the upload, but I was thinking if the internet cuts out or stays down to try to notify someone to go look into the upload again.

Long story short -  I could make this all happen if I could easily get the total # of parts being uploaded.

**Comment by bikeath1337** [comment]
Unless I misunderstand, you don't need an evaporate enhancement to get the number of parts to upload. The formula is here:

https://github.com/TTLabs/EvaporateJS/blob/master/evaporate.js#L1068

```javascript
partsCount = Math.ceil(file.size / partSize) || 1;
```

**Comment by bikeath1337** [comment]
@bdelamotte did this help at all?

**Comment by bdelamotte** [comment]
Hey @bikeath1337, sorry for the late reply. Yes! This is exactly what I needed. Thank you very much!

**Comment by bikeath1337** [comment]
Thanks for letting me know!
